jetstream/workflows/run.yaml

```diff
@@ -17,17 +17,15 @@ spec:
       parameters:
         - name: experiments
     steps:
-    - - name: analyse-experiment
-        template: analyse-experiment  
+    - - name: analyse-and-export
+        template: analyse-and-export
         arguments:
           parameters:
           - name: slug
             value: "{{item.slug}}"
           - name: date
             value: "{{item.date}}"
         withParam: "{{inputs.parameters.experiments}}"  # process these experiments in parallel
-        continueOn:
-          failed: true
```

Comment:
what are the implications of removing this?

Reply:
If the analysis fails, the export isn't executed. Which should be fine, running the export when analysis fails wouldn't upload any new data that isn't already in GCS.

Previously, `continueOn: failed` was necessary because if one experiment analysis failed, none of the export tasks (even for experiments where analysis executed successfully) would get run.